sources/packages/web/src/views/institution/locations/programs/LocationPrograms.vue

```diff
@@ -1,10 +1,7 @@
 <template>
   <full-page-container>
     <template #header>
-      <header-navigator
-        :title="locationDetails?.locationName"
```

Comment:
can you check, if the api was returning property `locationName`only for this purpose? in that case. you have remove the location name from the API as well as from the DTOs.
![image](https://user-images.githubusercontent.com/77353155/186502426-7d872b1b-e8e5-41b5-a613-96b751421fe9.png)

Reply:
Does the vue application not need the api to return the locationName attributes in the response payload to save in the store ?

Reply:
@ann-aot This API has been used in other places as well. Hence we can leave the API as is. And just modify the locationName to come from state here.

![image](https://user-images.githubusercontent.com/54600590/186537593-3736e731-aafe-4e0a-a849-baf196c1b040.png)

Reply:
my comment was to check if it's used anywhere else. Thanks

Reply:
Oh okay. I just checked using Ctrl + Shift + F in vs code. It was not used in any other place.

Reply:
I didn't get you @henryoforeh-aot. What is not used in any other place? what did you search.?

Reply:
@ann-aot @dheepak-aot  Please disregard my comment.